app/serializers/search/postgres_user_serializer.rb

```diff
@@ -0,0 +1,17 @@
+module Search
+  # to be renamed when ES is gone
+  class PostgresUserSerializer < ApplicationSerializer
```

Comment:
NOTE: this is smaller than the ES serialized data. Most of the things that are serialized by the server aren't used to build the UI so I proceeded backwards. By adding one field at the time :D